ipatests/test_integration/test_commands.py

```diff
@@ -679,3 +680,58 @@ def test_user_mod_change_capitalization_issue5879(self):
         assert 'Modified user "%s"' % (ipauser) in cmd.stdout_text
         assert 'First name: %s' % (modfirst) in cmd.stdout_text
         assert 'Last name: %s' % (modlast) in cmd.stdout_text
+
+    def test_sss_ssh_authorizedkeys(self):
+        """
+        When there is ssh key is added for IPA user then
+        'sss_ssh_authorizedkeys ipauser' should return ssh key.
+
+        Test for : https://pagure.io/SSSD/sssd/issue/3937
+
+        Steps:
+        1. add ipa user with ssh-key.
+        2. call sss_ssh_authorizedkeys and ssh key should be returned
+        """
+        user = 'testsshuser'
+        ssh_key_pair = tasks.generate_ssh_keypair()
+        pem_file = tempfile.mkdtemp(suffix='cert.pem', dir=paths.TMP)
+        key = tempfile.mkdtemp(suffix='private.key', dir=paths.TMP)
+        openssl_cmd = [
+            'openssl', 'req', '-x509', '-newkey', 'rsa:2048', '-days', '365',
+            '-nodes', '-keyout', key, '-out', pem_file, '-subj', '/CN=' + user]
+        self.master.run_command(openssl_cmd)
+        cert_b64 = self.get_cert_base64(self.master, pem_file)
+        try:
+            binary = '/usr/libexec/sssd/p11_child'
+            backup = tasks.FileBackup(self.master, binary)
```

Comment:
If this line fails then  `backup.restore()` in `finally` block will fail. It is better to place this line before of try-finally block